source/posix/thread.c

```diff
@@ -469,12 +473,15 @@ int aws_thread_current_name(struct aws_allocator *allocator, struct aws_string *
 #define THREAD_NAME_BUFFER_SIZE 256
 int aws_thread_name(struct aws_allocator *allocator, aws_thread_id_t thread_id, struct aws_string **out_name) {
     *out_name = NULL;
-#if defined(AWS_PTHREAD_GETNAME_TAKES_2ARGS) || defined(AWS_PTHREAD_GETNAME_TAKES_3ARGS)
+#if defined(AWS_PTHREAD_GETNAME_TAKES_2ARGS) || defined(AWS_PTHREAD_GETNAME_TAKES_3ARGS) || \
```

Comment:
```suggestion
#if defined(AWS_PTHREAD_GETNAME_TAKES_2ARGS) || defined(AWS_PTHREAD_GETNAME_TAKES_3ARGS) ||                            \
```
this whitespace diffierence is what's causing the "lint" to fail